src/components/table.js

```diff
@@ -16,30 +16,28 @@ function Table(props) {
 
   useEffect(()=>{
     if (props.summary===true) {
-      setStates(props.states.slice(0, 9));
+      setSortedStates(props.states.slice(0, 9));
     } else {
-      setStates(props.states);
+      setSortedStates(props.states);
     }
   }, [props.states]);
 
   useEffect(() => {
-    if(props.states[0]) {
+    if (props.states[0]) {
       setRevealedStates(props.states.reduce((a, state) => {
         return ({...a, [state.state]: false});
       }, {}));
     }
   }, [props.states]);
 
   useEffect(()=>{
-    if (states.length>0) {
-      let length = 0;
-      
-      props.states.map((state, i) => {
-        if (i!==0 && state.confirmed>0) length+=1;
-        if (i===props.states.length-1) setCount(length);
-      });
+    if (props.states.length) {
+      const affectedStates = props.states.reduce((pv, cv, ci) => {
+        return cv.confirmed > 0 && ci > 0 ? pv + 1 : pv;
+      }, 0);
+      setCount(affectedStates);
     }
-  }, [states.length]);
+  }, [props.states.length]);
 
   useEffect(()=>{
     getDistricts();
```

Comment:
Can you also remove the 1 in the inputs array here? It does not matter at all.

Reply:
Fixed it. It was a miss.